processing/data_collection/gazette/spiders/sp_presidente_prudente.py

```diff
@@ -8,7 +8,7 @@
 class SpPresidentePrudenhteSpider(BaseGazetteSpider):
```

Comment:
**nitpick** There is a type on class name. It should be `SpPresidentePrudenteSpider`.

Reply:
I will send a commit to fix this typo.

Reply:
Not so nitpick, but I should **definitely** have created my own branch for this PR. Noob here.